src/Behat/Gherkin/Loader/DirectoryLoader.php

```diff
@@ -57,6 +58,9 @@ public function supports($resource)
     public function load($resource)
     {
         $path = $this->findAbsolutePath($resource);
+        if ($path === false) {
+            throw new \LogicException("Unable to locate absolute path of supported resource: $resource");
```

Comment:
I hope that what is happening here is clear - it should not be possible for this exception to be thrown unless something happens between calling `supports()` and `load()`.
In any case, feel free to come up with a better message.. I'm a bit unsure with the current one.

Reply:
Or I guess if you call it yourself from somewhere without calling `supports` first - we don't know it's definitely been checked.

I might introduce a parent method `requireAbsolutePath($resource): string` to centralise / simplify this logic in the implementations.

And for a message maybe more like: "Cannot find absolute path for '$resource': filesystem may have changed during execution or `supports()` may not have been called"?

Reply:
I rewrote this part with the other suggestion (centralizing the check), so that message is no longer fully relevant. Feel free to comment on the change instead.